### DISCUSSION TITLE
Question about listen_server_address

### DISCUSSION THREAD

**Discussion by chip131001** [nanosonde/sip2rtsp#11] [Q&A]
Hello nanosonde,
I built and started the container as described.
I'm not sure what IP address I need to enter to add the camera to Scrypted?

**Comment by nanosonde** [comment]
Just leave it as it is. `listen_server_address: 0.0.0.0`. It means that the ONVIF HTTP server will listen on all interfaces.
More important is this `advertised_server_address: 10.10.10.70`. You must change it to the host IP address if you are running the SIP2RTSP in docker host networking mode. This is the IP address that will be used as string in the generated ONVIF HTTP responses that contain any UR pointing to the SIP2RTSP ONVIF HTTP server.
